src/api/app/views/webui/requests/change_devel_projects/new.html.haml

```diff
@@ -0,0 +1,22 @@
+- @pagetitle = 'Change Devel Project Request'
```

Comment:
`Request Devel Project Change` to be consistent with the links.

Reply:
Also capitalized.